src/csvcubed/utils/csvdataset.py

```diff
@@ -306,15 +306,28 @@ def _melt_data_set(
     ]
     id_cols = list(set(data_set.columns) - set(value_cols))
 
+    # Checking for any columns with the title "Value" and changing the value_name
+    # parameter passed to the melt function to "Not-Value" so that we don't
+    # trigger a pandas ValueError.
+    value_name = "Value"
+    if "Value" in value_cols:
```

Comment:
Does the pandas ValueError trigger if the column title contains "Value" (e.g. `if "Value" in`), or only if the title is "Value" exactly (== "Value)? I am assuming it is the latter, since this seems to work and I guess it wouldn't otherwise, but I am just checking because I don't know what the error looks like.

Reply:
I don't think you need to set value_name to "Value", you can just go straight to checking if "Value" is in value_cols

Reply:
@NickPapONS - Changed to loop through column titles in `value_cols` instead of only matching `"Value" in ..`

@SarahJohnsonONS - I think we need to assign a default to `value_name` because its only if `Value` is a title in `value_cols` do we assign a new string to `value_name` but we pass `value_name` regardless to `melt()`